### PR TITLE
Add an occ command to report updates

### LINKED ISSUE #8281: Add an occ command to report if updates are available
Right now, the option you have from the command line is to run updater.phar and perform any updates. There is no option to only check and report back any updates. An alternative is to have some bash script that constructs the complex url for update checks that update.phar uses or parse webpages and try to extract the information you need. 

It would be handy to have a command like `php occ update:check` to report back the available updates.

### PR DESCRIPTION
**Pull request by LEDfan**
Fixes #8281
Signed-off-by: Tobia De Koninck <LEDfan@users.noreply.github.com>

### CODE REVIEW

File: apps/updatenotification/lib/Command/Check.php [hunk 1]
```diff
@@ -0,0 +1,62 @@
+<?php
+
```
Comment (nickvergessen) [inline apps/updatenotification/lib/Command/Check.php]: license missing

File: apps/updatenotification/lib/Command/Check.php [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+		// Apps
+		$apps = $this->appManager->getInstalledApps();
+		foreach ($apps as $app) {
+			$update = $this->installer->isUpdateAvailable($app);
+			if ($update !== false) {
+				$output->writeln('Update for ' . $app . ' to version ' . $update . ' is available.');
+			}
+		}
+
+		return 0;
```
Comment (nickvergessen) [inline apps/updatenotification/lib/Command/Check.php]: I would either print a green message:
```
$output->writeln('<info>Everything up to date</info>');
```
or a yellow one:
```
$output->writeln('<comment>X updates available</comment>');
```
here as a summary

**Review by nickvergessen — APPROVED** [review]

**Comment by blizzz** [comment]
could also evaluate whatsNew information and link to the changelog. $r contains all info.

**Review by ChristophWurst — APPROVED** [review]
Code looks good. I like this feature :+1:

**Comment by blizzz** [comment]
failing tests unrelated & hiccups

**Pull request merged**